pom.xml

```diff
@@ -105,6 +105,18 @@
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>commons-lang</groupId>
+			<artifactId>commons-lang</artifactId>

```

Comment:
Please use latest versions as much as possible. In particular commons-lang3 supersedes commons-lang

https://mvnrepository.com/artifact/org.apache.commons/commons-lang3